scripts/portstat

```diff
@@ -555,8 +555,8 @@ class Portstat(object):
             print(table_as_json(table, header))
         else:
             print(tabulate(table, header, tablefmt='simple', stralign='right'))
-        if multi_asic.is_multi_asic(): 
-            print("\nReminder: Please execute 'show interface counters -d all' to include internal links\n")
+            if multi_asic.is_multi_asic(): 
```

Comment:
Question: Is this message only specific to multi_asic ? 

Reason I ask is that there are internal ports on single asic systems as well, like the recycle ports and inband ports. I assume we do want make sure those are displayed in single asic systems as well?

Reply:
good point. 
Was thinking about single-asic duts but not single-asic linecards for chassis, then I'll remove the if conditio for multi-asic and let it print for any type

Reply:
@kartik-arista I'll leave the multi_asic check as is for now, the original change is mainly for multi-asic prod devices, single-asic single-dut should not see this as they don't have internal links

Reply:
@wenyiz2021 - We do have ports like inband and recycle ports that are not part of the front panel on VOQ chassis liencards.  It does not matter whether these are single or multi-asic linecards. I just want to make sure that the command will work consistently in both these scenarios.

Reply:
thanks, I added print for chassis as well